flyway-core/src/test/resources/migration/dbsupport/postgresplus/sql/package/~V3__ComplexComments.sql

```diff
@@ -0,0 +1,95 @@
+--
```

Comment:
This file is "disabled" because it triggers a bug in the EDB JDBC drivers - EDB knows about it, and I expect will fix the bug sometime in the near-to-somewhat-near future. The test itself is valid, once the bug is fixed.